.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,70 @@
+name: Trivy Security Scanner
+on:
+  push:
+    branches:
+      - 6-22.04
+  pull_request:
+jobs:
+  build:
+    name: Build rock
+    uses: canonical/data-platform-workflows/.github/workflows/build_rock.yaml@v29.0.1
+  scan:
+    name: Trivy scan and SBOM Generation
+    needs: build
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Install required dependencies
+        run: |
+          # rockcraft
+          sudo snap install rockcraft --classic --edge
+
+          # yq
+          sudo snap install yq
+
+      - name: Download rock package(s)
+        uses: actions/download-artifact@v4
+        with:
+          pattern: ${{ needs.build.outputs.artifact-prefix }}-*
+          merge-multiple: true
+
+      - name: Import locally
+        run: |
+          sudo rockcraft.skopeo   --insecure-policy \
+              copy \
+              oci-archive:charmed-mongodb_6.0.6-2_amd64.rock \
```

Comment:
The name should come from the downloaded file I think ?
Otherwise this will break when we update the rock version